### PR TITLE
chore(examples): add cron job to update examples

### PR DESCRIPTION
**Pull request by korniltsev**
This will create PR's like this https://github.com/korniltsev/pyroscope/pull/7/files

### CODE REVIEW

File: .github/workflows/update-examples-cron.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          persist-credentials: false
+      - run: |
+          make tools/update_examples
+          if ! git diff --exit-code;
+          then
+            make tools/update_examples_pr
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (simonswine) [inline .github/workflows/update-examples-cron.yml:26]: Unsure if this is the right token, this does something very similar: https://github.com/grafana/pyroscope/blob/9b5c6d7f8c7eadf9bcb596c4b0ffc3f5b5607ddb/.github/workflows/update-contributors.yml#L32

**Review by simonswine — APPROVED** [review]
LGTM

The hard part remains the testing if it still compile/runs/sends profiles. When I ran this locally I mostly ran the one tracing example that contained all of the SDK, but we are breaking that up 😂

**Comment by aleks-p** [comment]
> The hard part remains the testing if it still compile/runs/sends profiles. When I ran this locally I mostly ran the one tracing example that contained all of the SDK, but we are breaking that up 😂

Technically we can still maintain a setup with all SDKs together. I don't think it is the best example to show users, but we could convert it to an integration test of some sort (outside of this PR).

**Comment by korniltsev** [comment]
> compile/runs/sends profiles

It would be nice if we could automate this somehow

**Pull request merged**